valohai_yaml/utils/upload_store.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from valohai_yaml.excs import ValidationError
+
+
+def parse_upload_store(upload_store: Optional[str]) -> Optional[str]:
```

Comment:
I'm not sure we need this "parsing" (actually validation) in valohai-yaml at all. Do we have precedent for doing this for e. g. the environment field (which is similar in purpose to this new field)?

Reply:
Why though?

Reply:
I thought it might be done on roi. But I wrote validation via maxLength in schema validation